engine/xml-config/src/main/java/org/datacleaner/configuration/DatastoreXmlExternalizer.java

```diff
@@ -334,6 +334,11 @@ public Element toElement(ElasticSearchDatastore datastore) {
         appendElement(ds, "port", datastore.getPort());
         appendElement(ds, "cluster-name", datastore.getClusterName());
         appendElement(ds, "index-name", datastore.getIndexName());
+        appendElement(ds, "username", datastore.getUsername());
+        appendElement(ds, "password", encodePassword(datastore.getPassword()));
+        appendElement(ds, "ssl", datastore.getSsl());
+        appendElement(ds, "keystorePath", datastore.getKeystorePath());
```

Comment:
Once .xsd is updated, this and the below string should also be updated of course.